source/_docs/automation/templating.markdown

```diff
@@ -3,7 +3,26 @@ title: "Automation Trigger Variables"
 description: "List all available variables made available by triggers."
 ---
 
-Automations support [templating](/docs/configuration/templating/) in the same way as scripts do. In addition to the [Home Assistant template extensions](/docs/configuration/templating/#home-assistant-template-extensions) available to scripts, the `trigger` template variable is available.
+Automations support [templating](/docs/configuration/templating/) in the same way as scripts do. In addition to the [Home Assistant template extensions](/docs/configuration/templating/#home-assistant-template-extensions) available to scripts, the `trigger`, `context`, and `this` template variables are available.
```

Comment:
This now became incorrect as the `context` has been added (which is out of scope for your PR a bit).

The `context` isn't always available and depends on the trigger platform. For example, when a state trigger fires, the context is part of the from and to properties.

I suggest removing that part from this PR.